include/libcgi/cgi_types.h

```diff
@@ -71,9 +71,9 @@ extern "C" {
 #endif
 
 /**
- *	General purpose linked list. Actually isn't very portable because
- *	uses only 'name' and 'value' variables to store data. Probably, in
- *	a future release, this will be replaced by another type of struct.
+ * General purpose linked list. Actually isn't very portable because
+ * uses only 'name' and 'value' variables to store data. Probably, in
+ * a future release, this will be replaced by another type of struct.
```

Comment:
This also looks like whitespace change only. Whitespace changes should go to separate commits.